_data/meltano/utilities/airflow/meltano.yml

```diff
@@ -0,0 +1,81 @@
+name: airflow
+namespace: airflow
+label: Airflow
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/meltano/airflow-ext
+pip_url: git+https://github.com/meltano/airflow-ext.git@main apache-airflow==2.3.3 --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.3.3/constraints-no-providers-3.8.txt
+executable: airflow_invoker
+variant: meltano
+maintenance_status: active
+commands:
+  describe:
+    executable: airflow_extension
+    args: describe
+  initialize:
```

Comment:
This might still change to plain `init` before merge (re: https://github.com/meltano/edk/discussions/23)